api/proto/teleport/workloadidentity/v1/attrs.proto

```diff
@@ -77,6 +77,14 @@ message WorkloadAttrsPodmanPod {
   map<string, string> labels = 2;
 }
 
+// Attributes sourced from the Systemd workload attestor.
+message WorkloadAttrsSystemd {
+  // Whether the workload passed Systemd attestation.
+  bool attested = 1;
```

Comment:
Will the `WorkloadAttrsSystemd` exist at all if it was not attested?
Sorry, if this is a dumb question. Trying to understand how it works when compared to other attestations.

Reply:
Not a dumb question at all! As with all of the attestors, if attestation fails (or the attestor isn't enabled) the whole attribute struct will be `nil`.

The `attested` bool flag is handy in WorkloadIdentity rules, though, because it enables you to say:

```yaml
rules:
  allow:
    - expression: workload.systemd.attested
```